### ISSUE TITLE
read/write access inside rar files

### ISSUE THREAD

**Issue by anmac1789** [hasse69/rar2fs#183]
Can this do read-write of rar archives when mounted in a folder on zorin OS ?

**Comment by hasse69** [comment]
Not sure I completely understand the question. If it is about if rar2fs should be possible to run on Zorin OS then the answer is yes. But if it is about write support the answer is no, which has nothing to do with what OS it is but rather because the RAR license prevents it.

**Comment by anmac1789** [comment]
> Not sure I completely understand the question. If it is about if rar2fs should be possible to run on Zorin OS then the answer is yes. But if it is about write support the answer is no, which has nothing to do with what OS it is but rather because the RAR license prevents it.

what i mean is can i make a folder or copy somwthing in a mounted rar archive ?

**Comment by hasse69** [comment]
No, it is not completely open source, see section 2 in the UnRAR license. There is no support library that supports RAR archive creation for this reason.

https://github.com/pmachapman/unrar/blob/master/license.txt
